eng/mgmt/AzSdk.test.reference.props

```diff
@@ -3,8 +3,8 @@
 		<TargetFrameworks>netcoreapp2.0</TargetFrameworks>
 	</PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Test.HttpRecorder" Version="[1.13.3, 2.0.0)" />
-    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="[1.7.6, 2.0.0)" />
+    <PackageReference Include="Microsoft.Azure.Test.HttpRecorder" Version="[1.13.1, 2.0.0)" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="[1.7.5, 2.0.0)" />
```

Comment:
Whose responsibility is it to update these version numbers, and how often are they expected to change?

Reply:
Great question. Mostly they will be updated on-demand when someone needs a newer versions. However we are trying to come up with an update strategy for these to ensure we are staying up-to-date. We also have some static analysis tools that will flag any dependency that has security issues for example so we will update them when they get flagged.